content/tutorials/channel-rewind.textile

```diff
@@ -0,0 +1,226 @@
+---
+title: Historical data with Channel Rewind
+section: tutorials
+index: 32
+---
+
+The Ably Realtime service organizes the message traffic within applications into named channels. Channels are the “unit” of message distribution; clients attach to any number of channels to subscribe to messages, and every message published to a channel is broadcasted to all its subscribers. 
+
+h2(#context). Context
+
+If a subscriber was to ever go offline, Ably by default, will retain all messages published and intended for that subscriber for a maximum of two min. During these two minutes, Ably will continuously try to re-establish the connection with that subscriber client, and if successful it'll relay all the retained messages to the subscriber in the order of their arrival. However, some clients may expect longer disconnections and hence may need message retention for more than the default two minute duration. For such scenarios Ably provides a feature called "History":/core-features/history using which one can retrieve missed historical messages that are a maximum of 72 hours old.
```

Comment:
I had quite a few detailed comments on this section but, before dealing with them, I think we need to fix the story.

Resume is peripherally relevant to rewind, but I think this section needs to start by motivating the rewind functionality. The discussion about resume and history is not relevant.

So I think the narrative for this needs to be something like:

- usually in pub/sub applications a client is interested in receiving messages as they occur in real time. Ably's API reflects this, in that an application can `subscribe()` to a channel; the client attaches to the channel, and receives messages from that point forward;

- however, in some applications it is useful to have access also to messages that occurred in the past. Ably supports a `history()` API whereby, for channels that are persisted, it is possible to query explicitly for messages on a channel within any specified time bounds;

- for some applications, the requirement for historical access is simply to be able to obtain the most recent message or messages on a channel. This can apply, for example, if <example ...>

- it would be possible for an application to achieve that by subscribing to a channel and also using the history API. However, Ably now provides a much simpler way to subscribe to a channel from a specified point in the past; that is, the subscriber will be sent all historical messages on the channel from the specified point forwards, and will receive all subsequent messages in realtime.

Reply:
Fixed, thanks.